### PR TITLE
support for className

### PR DESCRIPTION
**Pull request by wuct**
This PR is for #6 .

### CODE REVIEW

File: src/MorphTransition.js [hunk 1]
```diff
@@ -36,8 +36,9 @@ class MorphTransition extends React.Component {
     render() {
         var width = this.props.width;
         var height = this.props.height;
+        var className = this.props.className;
         return (
-            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height}>
+            <svg viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit width={width} height={height} className={className}>
```
Comment (wuct) [inline src/MorphTransition.js]: Maybe we should just use object destructuring here and let consumers pass any prop they like, ex: `style`, `onClick`.

```
<svg {...this.props} viewBox={this.viewBox} preserveAspectRatio="xMidYMid meet" fit>
```

Comment (gorangajic) [inline src/MorphTransition.js]: Yes let's pass all props, in native version it's already like that
https://github.com/gorangajic/react-svg-morph/blob/master/src/MorphTransitionNative.js#L15

**Comment by wuct** [comment]
@gorangajic I have modified the code to use desctructuring.

**Comment by gorangajic** [comment]
great, thanks

**Comment by wuct** [comment]
Thanks for merging

**Pull request merged**